app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -6,13 +6,13 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudV
   end
   supports_not :snapshot_create
   supports_not :update
-  supports :attach do
-    unsupported_reason_add(:attach, _("the volume is not connected to an active Provider")) unless ext_management_system
-    unsupported_reason_add(:attach, _("cannot attach non-shareable volume that is in use.")) if status == "in-use" && !multi_attachment
+  supports :attach_volume do
+    unsupported_reason_add(:attach_volume, _("the volume is not connected to an active Provider")) unless ext_management_system
+    unsupported_reason_add(:attach_volume, _("cannot attach non-shareable volume that is in use.")) if status == "in-use" && !multi_attachment
   end
-  supports :detach do
-    unsupported_reason_add(:detach, _("the volume is not connected to an active Provider")) unless ext_management_system
-    unsupported_reason_add(:detach, _("the volume status is '%{status}' but should be 'in-use'") % {:status => status}) unless status == "in-use"
+  supports :detach_volume do
+    unsupported_reason_add(:detach_volume, _("the volume is not connected to an active Provider")) unless ext_management_system
+    unsupported_reason_add(:detach_volume, _("the volume status is '%{status}' but should be 'in-use'") % {:status => status}) unless status == "in-use"
```

Comment:
The attach / detach features no long need the `_volume` suffix. See #373 for details